packages/nimbus/src/components/toggle-button-group/toggle-button-group.mdx

```diff
@@ -12,223 +12,442 @@ menu:
   - ToggleButtonGroup
 tags:
   - component
+figmaLink: >-
+  https://www.figma.com/design/AvtPX6g7OGGCRvNlatGOIY/NIMBUS-design-system?node-id=2693-2416&m=dev
 ---
 
-# ToggleButtonGroup
+# Button Group
 
-The ToggleButtonGroup component provides a way to group multiple toggle buttons
-together, visually and functionally. It's built on top of React Aria's
-ToggleButtonGroup for accessibility and offers a compound component API for
-flexibility.
+---
+
+A set of closely related, mutually exclusive or complementary actions that are important enough to be displayed directly in the interface for quick access.
+
+## Overview
+
+Button groups can be used several ways, with and without icons, and spanning from two selections to several.
+
+### Resources
+
+Deep dive on details and access design library.
+
+[Figma library](https://www.figma.com/design/AvtPX6g7OGGCRvNlatGOIY/NIMBUS-design-system?node-id=2693-2416&m=dev)
```

Comment:
we should add links to the underlying react-aria component:
https://react-spectrum.adobe.com/react-aria/ToggleButtonGroup.html#togglebuttongroup

and the relevant w3c aria pattern:
https://www.w3.org/WAI/ARIA/apg/patterns/toolbar/

Reply:
Thanks, addressed in c4d2117ef1e7098eb2e28ae88ce1a398e258964d